rust/src/fvm/blockstore/cgo.rs

```diff
@@ -34,13 +34,16 @@ impl Blockstore for CgoBlockstore {
             match cgo_blockstore_has(self.handle, k_bytes.as_ptr(), k_bytes.len() as i32) {
                 // We shouldn't get an "error not found" here, but there's no reason to be strict
                 // about it.
-                0 | ERR_NOT_FOUND => Ok(false),
+                0 => Ok(false),
+                x if x == FvmError::NotFound as i32 => Ok(false),
```

Comment:
Nitpick (this doesn't prevent merging and could be done later on). In this match the thing it matches on is quite often bound to a variable. I think binding it directly would make the code easier to read. Especially since the match is a function (in the cases below with quite a few parameters). It would also reduce the scope of `unsafe`. I propose something like:

```rust
let has = unsafe { cgo_blockstore_has(self.handle, k_bytes.as_ptr(), k_bytes.len() as i32) } as i32;
match has {
    // We shouldn't get an "error not found" here, but there's no reason to be strict
    // about it.
    0 => Ok(false),
    FvmError::NotFound => Ok(false),
    1 => Ok(true),
    // Panic on unknown values. There's a bug in the program.
    2.. => panic!("invalid return value from has: {}", has),
    // Panic if the store isn't registered. This means something _very_ unsafe is going
    // on and there is a bug in the program.
    FvmError::InvalidHandle => {
        panic!("blockstore {} not registered", self.handle)
    }
    // Otherwise, return "other". We should add error codes in the future.
    _ => Err(anyhow!("cgo blockstore 'has' failed with error code {}", has)),
}
```

Though it's a matter of style and not correctness, so feel free to ignore.

Reply:
I agree this code could be cleaned up, I was going for minimal changes in here this time around 😅  given the amount of code this PR already changes.